build.gradle

```diff
@@ -131,6 +134,13 @@ task sourcesJar(type: Jar, dependsOn: classes) {
 	from sourceSets.main.allSource
 }
 
+shadowJar {
+	baseName = 'shadow'
+	classifier = ''
+	archiveVersion = ''
+	relocate("com.google.gson", "net.fabricmc.loader.libraries.gson")
```

Comment:
Isn't our main purpose to relocate guava and make mixin/jimfs use relocated guava?